### ISSUE TITLE
Cerner deprecation: update code that is referencing deprecated selectors

### ISSUE THREAD

**Issue by jilladams** [department-of-veterans-affairs/va.gov-cms#12822]
## Description
From https://github.com/department-of-veterans-affairs/vets-website/pull/23036#issuecomment-1448615887: 

1. **Unit tests** are still referencing a selector that has been removed, so the references are failing during build:
`src/platform/user/tests/selectors.unit.spec.js`- PW should check for and remove any tests that reference selectors that have been removed here: https://github.com/department-of-veterans-affairs/vets-website/pull/23036/files#diff-9ec1d9d6f4976449cb9b5f337f1946801ecb0732d3f216d922ab2f790db5f073L27

2. References in a few different exportsFile.js files refer to a number of things that have been removed:
`src/platform/user/exportsFile.js` - PW should make this change and we could do in [this PR](https://github.com/department-of-veterans-affairs/vets-website/pull/23036). We are not codeowners of the file.
`src/platform/utilities/exportsFile.js` - same
We need to check for any references to: 
```
  CERNER_APPOINTMENTS_BLOCKLIST,
  CERNER_FACILITY_IDS,
  CERNER_MEDICAL_RECORDS_BLOCKLIST,
  CERNER_MESSAGING_BLOCKLIST,
  CERNER_RX_BLOCKLIST,
  CERNER_TEST_RESULTS_BLOCKLIST,
```
## Eng notes
Code to satisfy this ticket should be based on and can be pushed directly to the active branch for #9307: [9307-remove-hardcoded-cerner-api](https://github.com/department-of-veterans-affairs/vets-website/tree/9307-remove-hardcoded-cerner-api). Or: can PR to that branch, if you'd like separate code review, not required.

## Acceptance Criteria
- [ ] Unit tests do not reference removed code.
- [ ] Unit tests pass (TBD Ryan's confirmation that this is possible without the VAOS / Personalization / Health products downstream changes as well)
- [ ] exportsFile.js no longer references removed code

### Team
Please check the team(s) that will do this work.

- [ ] `CMS Team`
- [X] `Public Websites`
- [ ] `Facilities`
- [ ] `User support`

**Comment by jilladams** [comment]
Tagged with Facilities in order for this to appear in the Facilities sprint board, as we've requested some of Ryan's time to work on it in Sprint 79. Work should only be counted in the PW sprint report.

**Comment by jilladams** [comment]
Ryan merged changes directly to #9307 branch. Closing, and if we find after 9307 other dependencies are resolved from downstream teams that we need other unit test, etc, updates, we will ticket separately. Closing.
